.travis.yml

```diff
@@ -1,9 +1,18 @@
+language: generic
 sudo: required
 
 services:
   - docker
 
+env:
+  - FEDORA=27
+  - FEDORA=28
+  - FEDORA=29
+  - FEDORA=rawhide
+
 install:
+  - sed -i "s/fedora-28-x86_64/fedora-${FEDORA}-x86_64/" mock.cfg
+  - sed -i "s|FROM fedora|FROM registry.fedoraproject.org/fedora:${FEDORA}|" Dockerfile
```

Comment:
Can it be changed directly in the files?

Reply:
It is. I probably don't understand the question.

The `registry.fedoraproject.org/` part is there for f29 to work before the release. I can add it directly to the file but it would make no difference.